openfecwebapp/templates/raising-breakdown.html

```diff
@@ -20,18 +21,70 @@ <h1 class="icon-heading--raising-circle">{{ title }}</h1>
   <a class="sidebar__title" href="{{ url_for('search', _anchor='raising') }}">&lsaquo; Raising overview</a>
   <ul class="sidebar__content js-toc">
     <li class="side-nav__item"><a class="side-nav__link" href="#top-raisers">Who's raising the most</a></li>
-    <li class="side-nav__item"><span class="side-nav__link is-disabled">Where contributions come from</span></li>
+    <li class="side-nav__item"><a class="side-nav__link" href="#by-state">Where contributions come from</a></li>
     <li class="side-nav__item"><span class="side-nav__link is-disabled">The size of contributions</span></li>
   </ul>
 </nav>
 {% endblock %}
 
 {% block sections %}
 <section id="raising-breakdowns">
+
   <section id="top-raisers" class="content__section">
     <h2>Who's raising the most</h2>
     {{ breakdowns.top_entities(top_raisers, coverage_start_date, coverage_end_date, cycle, 'raised', top_category, pagination ) }}
   </section>
+
+  <section id="by-state" class="content__section--extra content__section--ruled js-breakdown-map" data-cycle="{{ cycle }}">
+    <h2>Where contributions come from</h2>
+    <form action="" method="GET">
+      <h3 class="u-no-margin">
+        <label for="top-category" class="breakdown__title label t-inline-block">A state-by-state breakdown of individual contributions from each state to </label>
+        <select id="top-category" name="top_category" class="js-category form-element--inline" aria-controls="top-table">
```

Comment:
This needs to have a different ID, because this is shared by the top list above.